lib/manageiq/providers/workflows/engine.rb

```diff
@@ -53,13 +66,13 @@ def self.set_floe_runner
             }.merge(floe_runner_settings.kubernetes)
 
             Floe.set_runner("docker", "kubernetes", options)
-          elsif MiqEnvironment::Command.is_appliance? || MiqEnvironment::Command.supports_command?("podman")
+          when "podman"
             options = {}
             options["root"] = Rails.root.join("data/containers/storage").to_s if Rails.env.production?
             options.merge!(floe_runner_settings.podman)
 
             Floe.set_runner("docker", "podman", options)
-          else
+          else # "docker"
```

Comment:
@kbrock maybe overkill but what do you think about `when "docker"` and the `else` condition handling if you entered an invalid runner name in settings?  The way this is, if you typo'd `pdoman` it'd default to docker.